pkg/render/logstorage/esgateway/esgateway.go

```diff
@@ -0,0 +1,324 @@
+package esgateway
+
+import (
+	"fmt"
+	"strings"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	operatorv1 "github.com/tigera/operator/api/v1"
+	"github.com/tigera/operator/pkg/components"
+	"github.com/tigera/operator/pkg/render"
+	relasticsearch "github.com/tigera/operator/pkg/render/common/elasticsearch"
+	rmeta "github.com/tigera/operator/pkg/render/common/meta"
+	"github.com/tigera/operator/pkg/render/common/secret"
+)
+
+const (
+	DeploymentName          = "tigera-es-gateway"
+	ServiceAccountName      = "tigera-es-gateway"
+	RoleName                = "tigera-es-gateway"
+	VolumeName              = "tigera-es-gateway-certs"
+	ElasticsearchUserSecret = "tigera-es-gateway-elasticsearch-access"
+	ServiceName             = "tigera-secure-es-gateway-http"
```

Comment:
We could just call it `tigera-es-gateway`

Reply:
I think we wanted all the names to be `tigera-secure`, not just `tigera`. Maybe that's changed though.